### DISCUSSION TITLE
Kayak is alive

### DISCUSSION THREAD

**Discussion by sauljabin** [Textualize/textual#2114] [Show and tell]
Hello folks, this is kayak https://github.com/sauljabin/kayak check it out. 

@willmcgugan

**Comment by willmcgugan** [comment]
Hi Saúl. That looks amazing. Just the kind of use-case we want to see.

I like what you've done with the colors. The more minimal look really works!

**Reply by sauljabin** [reply]
Thanks!. Guys you have improved the developer experience in a good way. It's easy to initiate a new project. And now there is no freezing when having concurrent tasks. 

Speaking of colors, I did this to set the css global variables, is it ok?
https://github.com/sauljabin/kayak/blob/main/kayak/styles/colors.py
https://github.com/sauljabin/kayak/blob/469151cf9d83b3fb3e5f0f0cb325a028c425f510/kayak/tui.py#L135

Speaking of css, what a clever feature, you have removed a lot of unnecessary python code. 
I did this for hiding stuff https://github.com/sauljabin/kayak/blob/main/kayak/tui.css#L147.

**Comment by TomJGooding** [comment]
I love the design and definitely something to aspire to! It gives me the "feel" of a TUI that other examples I've seen don't, if that makes sense? But maybe that's just the ASCII art logo!

Sorry this a bit off topic but I noticed you're using ruff - how are you finding it?

**Reply by sauljabin** [reply]
Very nice honestly. I have combined black, ruff and mypy and everything is working well. I've just started using ruff so I need to test it more. But, so far is faster and straight forward.

**Reply by sauljabin** [reply]
+ thanks for your feedback about kayak and how it looks like
